README.md

```diff
@@ -105,6 +105,9 @@ If you are interested in contributing to EKS Blueprints, see the [Contribution g
 
 ---
 
+> **_NOTE:_** Use terraform versions after version 1.0.0 and before version 1.3.0.  
```

Comment:
terraform-docs will wipe this out (after CI checks complain)

Reply:
it's before the `<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->` , you sure?

Reply:
oh .... I'm not that smart, please  ignore me 😬